game.rb

```diff
@@ -0,0 +1,15 @@
+require './item'
+
+class Game < Item
+  attr_accessor :mulitiplayer, :last_played_at
+
+  def initialize(mulitiplayer, last_played_at, *args)
+    @mulitiplayer = mulitiplayer
+    @last_played_at = last_played_at
+    super(*args)
+  end
+
+  def can_be_archived?
+    super && (Time.now.year - Date.parse(@last_played_at).year) > 2
+  end
+end
```

Comment:
- [ ] Each game should have a genre, author, and label. In your case, the game only possesses three attributes ie., multiplayer,  last played at, and published. Kindly update this class accordingly. 

- [ ] The games data is not being saved locally. Kindly fix this error.

Reply:
Kindly check what diagram require 
![image](https://user-images.githubusercontent.com/100236377/189278069-1abe1e63-03e6-4fc4-b065-b72d680e692b.png)
so you ask what diagram doesn't ask for game

Reply:
![image](https://user-images.githubusercontent.com/100236377/189288337-84d74f48-9e00-4b37-9b56-65ab09d8d0e9.png)

AND ALSO THE GAMES DATA IS SAVE LOCALY